bip-0068.mediawiki

```diff
@@ -25,7 +25,7 @@ The transaction nLockTime is used to prevent the mining of a transaction until a
 
 ==Specification==
 
-This specification defines the meaning of sequence numbers for transactions with an nVersion greater than or equal to 2 for which the rest of this specification relies on.
+This specification defines the meaning of sequence numbers for transactions with an nVersion greater than or equal to 2, or any negative version, for which the rest of this specification relies on.
```

Comment:
```suggestion
This specification defines the meaning of sequence numbers for transactions with an nVersion greater than or equal to 2 when interpreted as an unsigned little-endian integer, for which the rest of this specification relies on.
```

Reply:
leaving as-is unless people feel strongly